airbyte-ci/connectors/base_images/README.md

```diff
@@ -41,6 +41,7 @@ RUN mkdir /usr/share/nltk_data
 
 | Version | Published | Docker Image Address | Changelog | 
 |---------|-----------|--------------|-----------|
+|  2.0.0 | ❌|  | Use Python 3.10 |
```

Comment:
I'll publish it once https://github.com/airbytehq/airbyte/pull/41057 is merged and adopted first.